app/util/geo-utils.coffee

```diff
@@ -0,0 +1,11 @@
+toRad = (deg) -> deg * Math.PI / 180
+toDeg = (rad) -> rad * 180 / Math.PI
+
+getBearing = (lat1,lng1,lat2,lng2) ->
+  lonScale = Math.cos toRad (lat1+lat2)/2
+  dy = lat2-lat1
+  dx = (lng2-lng1)*lonScale
+  (toDeg(Math.atan2(dx, dy)) + 360) % 360
+
+module.exports =
+  getBearing: getBearing
```

Comment:
lint:

  ⚡ app/util/geo-utils.coffee
     ⚡ #4: Spaces are required after commas. ,.
     ⚡ #4: Spaces are required after commas. ,.
     ⚡ #4: Spaces are required after commas. ,.
     ⚡ #5: Operators must be spaced properly. +.
     ⚡ #5: Operators must be spaced properly. /.
     ⚡ #6: Operators must be spaced properly. -.
     ⚡ #7: Operators must be spaced properly. -.
     ⚡ #7: Operators must be spaced properly. *.